src/main/java/seedu/lifeasier/parser/Parser.java

```diff
@@ -54,7 +54,7 @@
 
     public static final String PARAM_CODE = "/code";
     public static final String PARAM_DATE = "/date";
-    public static final String PARAM_TIME = "/time";
+    public static final String PARAM_TIME = "/from";
```

Comment:
Could you consider renaming the parameter too?
```suggestion
    public static final String PARAM_FROM = "/from";
```